src/encoded/upgrade/dataset.py

```diff
@@ -731,3 +731,49 @@ def experiment_35_36(value, system):
     # https://encodedcc.atlassian.net/browse/ENCD-5964
     if value.get('assay_term_name') == 'Capture Hi-C':
         value['assay_term_name'] = 'capture Hi-C'
+
+
+@upgrade_step('reference', '20', '21')
+def reference_20_21(value, system):
+    # https://encodedcc.atlassian.net/browse/ENCD-6134
+    new_elements_selection_method = []
+    if 'elements_selection_method' in value:
+        for element_selection_method in value['elements_selection_method']:
+            if re.match('ATAC-seq', element_selection_method):
+                new_elements_selection_method.append('accessible genome regions')
+                if 'notes' in value:
+                    value['notes'] = f'{value.get("notes")}. This elements_selection_method was previously ATAC-seq, but now renamed to be accessible genome regions.'
+                else:
+                    value['notes'] = 'This elements_selection_method was previously ATAC-seq, but now renamed to be accessible genome regions.'
+            elif re.match('DNase-seq', element_selection_method):
+                new_elements_selection_method.append('DNase hypersensitive sites')
+                if 'notes' in value:
+                    value['notes'] = f'{value.get("notes")}. This elements_selection_method was previously DNase-seq, but now renamed to be DNase hypersensitive sites.'
+                else:
+                    value['notes'] = 'This elements_selection_method was previously DNase-seq, but now renamed to be DNase hypersensitive sites.'
+            elif re.match('GRO-cap', element_selection_method):
+                new_elements_selection_method.append('transcription start sites')
+                if 'notes' in value:
+                    value['notes'] = f'{value.get("notes")}. This elements_selection_method was previously GRO-cap, but now renamed to be transcription start sites.'
+                else:
+                    value['notes'] = 'This elements_selection_method was previously GRO-cap, but now renamed to be transcription start sites.'
+            elif re.match('point mutations', element_selection_method):
```

Comment:
You can combine checking for point mutations and SNPs since they are upgraded to the same thing